packages/css/src/components/button-row/button-row.scss

```diff
@@ -0,0 +1,20 @@
+.dso-button-row {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $u2;
+  margin: $u2 0 $u4;
+
+  .btn:not(.dso-list-button),
+  .dso-primary,
+  .dso-secondary,
+  .dso-tertiary {
+    white-space: nowrap; // stylelint-disable-line declaration-property-value-disallowed-list -- No word wrapping in buttons
```

Comment:
Over deze toelichting had @tfrijsewijk gesteld:
> Stylelint dwingt een beschrijving af om een toelichting te geven, niet uit te leggen wat er gebeurt.. Want dat vertelt de code ons al.

Maar ik kan eigenlijk ook geen betere motivatie verzinnen dan deze "No word wrapping in buttons".

Reply:
Ja klopt, en het is iets wat ook vaker gedaan wordt.

Reply:
Maar dan staat er (bijna altijd) een inhoudelijke toelichting:

![image](https://user-images.githubusercontent.com/9696261/165279856-f3f7303f-c06f-4d85-803c-f6a626a3ded7.png)

Reply:
Ok, dus @PieterVerheij probeer te verwoorden WAAROM deze `white-space: nowrap` zo nodig is (en dus: waarom we hier 'zondigen' tegen een vrije 'reflow'). Is dat bv. om het icoon er altijd naast te houden, of omdat het design nu eenmaal vereist dat (deze) buttons een-regelig blijven?